deploy/deployment/worker-dataset-deployment.yaml

```diff
@@ -20,7 +20,10 @@ spec:
           image: slinkythegraph/slinky
           imagePullPolicy: Always
           command: ["slinky"]
-          args: ["work", "dataset", "--prod"]
+          args: ["work", "dataset", "--debug"]
```

Comment:
I did this because most logs are debug and if anything breaks I'd like to see the most logs that I can instead of restarting it and trying to hit the error a second time with more logging. I'm pretty impartial though